datus/tools/mcp_tools/llm_native_mcp_handler.py

```
@@ -0,0 +1,345 @@
+            # Make HTTP POST to execute tool
+            response = requests.post(
+                execute_url,
+                json=arguments,
+                timeout=60,  # Codex uses 60s timeout
+                headers={'Content-Type': 'application/json'}
+            )
+
+            if response.status_code == 200:
+                return response.json()
+            else:
+                raise Exception(f"Tool execution failed: HTTP {response.status_code} - {response.text}")
```

Ruff (0.14.5)

Abstract `raise` to an inner function

(TRY301)